doc/devices/qvm.rst

```diff
@@ -190,36 +190,4 @@ After measuring the qubit state, we can determine the probability :math:`P_0` of
 
```

Comment:
Consider removing the `CPHASE`, `ISWAP` and `PSWAP` gates from the `Supported operations` section of this file, given that they are already supported in PennyLane.